components/ItemCard/index.js

```diff
@@ -0,0 +1,3 @@
+import { ItemCard } from "./ItemCard";
+
+export default ItemCard;
```

Comment:
Why do you have this file? 
Either you delete this file or you just paste everything from the ItemCard.js into this file and delete the ItemCard.js

Reply:
This is called a barrel file.
Usually you bundle multiple files together to export it with one file.
I'm using it to keep my `import` cleaner
Usually I work with index.js files, but if open multiple index.js files it kind looks messy in my VSCode Tab.

In this case I can just import files normally.
`import ItemCard from "@/components/ItemCard"`

Reply:
Kept the barrel function only for Icons
- [x] Removed barrel function for components